src/services/message.service.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseChatroomDto, BaseMessageDto, CreateMessageDto } from '@/dtos';
+import { Chatroom, Messages } from '@/entities/nosql';
+import { HttpException } from '@/exceptions';
+import { model, ObjectId } from 'mongoose';
+
+class MessageService {
+  public async getChatroomMessages(id: ObjectId): Promise<BaseMessageDto[]> {
```

Comment:
Can you explain what ObjectId from mongoose does?

Reply:
It is an ObjectId typing for typescript for mongodb documents.